src/web/routes/application/steps/address/select-address/select-address.js

```diff
@@ -35,11 +35,16 @@ const resetAddressState = (req) => {
 }
 
 const getAddressDataFromSession = (req, addressId) => {
-  const addresses = req.session.postcodeLookupResults.map(buildAddressOption(addressId))
+  const results = req.session.postcodeLookupResults.map(buildAddressOption(addressId))
+  const addressCount = {
+    value: '',
+    text: req.t('address.numberOfAddressesFound', { count: results.length }),
+    disabled: true,
+    selected: !results.some(addr => addr.selected)
+  }
+  const addresses = results.length === 0 ? [] : [ addressCount ].concat(results)
```

Comment:
This line could be shortened to:
```
const addresses = results.length ? [] : [addressCount, ...results]
```

Reply:
.. actually - wouldn't `results` return an empty array by default? So line 45 would just need to be:
```
const addresses = [addressCount, ...results]
```